regression/scripts/B737/mission_B737_segment_altitude_test.py

```diff
@@ -0,0 +1,476 @@
+# mission_B737_segment_altitude_test.py
```

Comment:
can we just change the B737 mission as it is?

Reply:
Sure. I left it in since I think the regressions are the most reliable place to get a running vehicle when starting a new project so I thought it might be helpful to leave it with a realistic mission. I can just make some small adjustments though.